lib/libutee/include/tee_api_defines.h

```diff
@@ -85,8 +91,13 @@
 #define TEE_LOGIN_APPLICATION           0x00000004
 #define TEE_LOGIN_APPLICATION_USER      0x00000005
 #define TEE_LOGIN_APPLICATION_GROUP     0x00000006
-#define TEE_LOGIN_KERNEL                0xA0000000
+#define RSVD_GP_DEFINED_LOGIN_START1    0x00000007
```

Comment:
Why do we need the `*RSVD_*` defines? They are not in the spec, a comment would be enough (and much clearer since I had to read the spec before I realized it means reserved).